src/html/index.html

```diff
@@ -10,118 +10,110 @@
+                            <li class="nav-menu__item"><a class="nav-menu__item__link" href="#">Portfolio</a></li>
+                            <li class="nav-menu__item"><a class="nav-menu__item__link" href="#">News</a></li>
+                            <li class="nav-menu__item"><a class="nav-menu__item__link" href="#">Contact</a></li>
+                        </ul>
+                    </nav>
+                </div>
             </div>
-        </div>
-        <!--End first section-->
-        <!--Star Second section-->
-        <div class="container">
-            <div class="row align-items-center">
+            <div class="row">
                 <div class="col-12 col-xs-12">
-                    <section class="intro">
-                        <h1 class="intro__title">"Simplicity is the ultimate sophistication."
-                            <span class="littlespan">- Leonardo da Vinci</span>
+                    <section class="title">
+                        <h1 class="title__maintitle">"Simplicity is the ultimate sophistication."
+                            <span class="title__maintitle__span">- Leonardo da Vinci</span>
                         </h1>
-                        <h2 class="intro__paragraph">In character, in manner, in style, in all things, the supreme excellence is simplicity.
-                             <a class="link" href="../min-res/minimalism_responsive.zip">Download the template</a>
+                        <h2 class="title__subtitle">In character, in manner, in style, in all things, the supreme excellence is simplicity.
+                            <a class="link" href="../min-res/minimalism_responsive.zip">Download the template</a>
                         </h2>
                     </section>
                 </div>
             </div>
         </div>
-        <!--End Second section-->
-        <!--Star third section-->
-        <div class="container">
+        <!--End header section-->
+        <!--Start body-content section-->
+        <div class=" body-content container">
             <div class="row align-items-center">
-                <div class="fourcol col-xs-12 col-md-12 col-xl-4">
+                <div class="col-xs-12 col-md-4 col-xl-4">
                     <section class="box">
                         <h3 class="box__title">About us</h3>
                         <p class="paragraph">Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec molestie. Sed aliquam sem ut arcu. Phasellus sollicitudin. 
                             Vestibulum condimentum  facilisis nulla. In hac habitasse platea dictumst. Nulla nonummy.</p>
-                        <a href="#"><img class="photo" src="images/picture1.jpg" width="343" height="305" alt="picture1"></a>
-                        <a class="button link" href="#"><span class="button__title">Read more</span></a>
+                        <a href="#"><img class="photo"  src="images/picture1.jpg" width="343" height="305" alt="picture1"></a>
```

Comment:
Don't use inline CSS rules (width, height). Is very confusing and has more priority than external CSS rules